### PR TITLE
[CALC] Add few units for conversion

### PR DESCRIPTION
**Pull request by Bishal-Bh**
## Purpose

This changes is for case study and making minor changes in this OS.

### CODE REVIEW

**Comment by binarymaster** [comment]
Can you please describe better what your change does? So I can apply correct labels.

And make sure to follow rules in https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md

File: base/applications/calc/convert.c [hunk 1]
```diff
@@ -75,6 +75,7 @@ static const conv_t conv_ANGLE[] = {
     DECLARE_CONV_UNIT(ANGLE, DEGREES,    "$",       "$")
     DECLARE_CONV_UNIT(ANGLE, GRADIANS,   "$*0.9",   "$/0.9")
     DECLARE_CONV_UNIT(ANGLE, RADIANS,    "$/P*180", "$*P/180")
+    DECLARE_CONV_UNIT(ANGLE, KONA,    "$/P*180", "$*P/180")
```
Comment (HBelusca) [inline base/applications/calc/convert.c:78]: What's this unit? I cannot find it documented on the internet.
Is it expected that it uses the same conversion formulae as the radians?

File: base/applications/calc/convert.c [hunk 2]
```diff
@@ -148,6 +149,7 @@ static const conv_t conv_AREA[] = {
     DECLARE_CONV_UNIT(AREA, SQUARE_YARD,            "$*0,83612736",      "$/0,83612736")
     DECLARE_CONV_UNIT(AREA, TAN,                    "$*1080000/1089",    "$*1089/1080000")
     DECLARE_CONV_UNIT(AREA, TSUBO,                  "$*1188/1089",       "$*1089/1188")
+    DECLARE_CONV_UNIT(AREA, AANA,                  "$*1188/1089",       "$*1089/1188")
```
Comment (HBelusca) [inline base/applications/calc/convert.c:152]: Is it the correct translation formula?

**Comment by Bishal-Bh** [comment]
I forgot to add the correct translation for aana.  aana= 31,79 m^2 and
there is no angle measurement called kona I just forgot to delete that line
of code.

**Comment by binarymaster** [comment]
@BisHal-Bha are you going to eventually revisit this?

**Pull request closed without merging**